app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/addEmployeeJobInfrmation/index.tsx

```
@@ -6,47 +6,96 @@ import WorkScheduleForm from '../../../../_components/allFormData/workScheduleFo
 import { useGetEmployee } from '@/store/server/features/employees/employeeDetail/queries';
 import BasicSalaryForm from '../../../../_components/allFormData/basickSalaryForm';
+import { useParams } from 'next/navigation';
 
 interface Ids {
   id: string;
   onInfoSubmition?: () => void;
+  onJobInfoUpdated?: () => void;
+  isNavBarModal?: boolean;
 }
 export const CreateEmployeeJobInformation: React.FC<Ids> = ({
   id: id,
```

ESLint

[error] 'id' is defined but never used.

(@typescript-eslint/no-unused-vars)